index.js

```diff
@@ -20,10 +20,7 @@ if (fs.existsSync('composer.json')) {
 
     if (!validationResult.valid) {
         validationResult.errors.forEach(function (outputUnit) {
-           core.error("There is an error in the keyword located by {0}: {1}".format(
```

Comment:
This is also deprecated but when I tried to add https://github.com/laminas/laminas-ci-matrix-action/pull/46/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R48-R50, I had issues with `String.format` and thus, I wanted to ensure that this will definitely work.